README.md

```diff
@@ -293,11 +293,11 @@ jobs:
           --resource=2ff814a6-3304-4ab8-85cb-cd0e6f879c1d \
           --query accessToken -o tsv)" >> $GITHUB_ENV
       - name: Trigger model training notebook from PR branch
-        uses: databricks/run-databricks-notebook@v0
+        uses: databricks/run-notebook@v0
         with:
           # Run our notebook against a remote repo
           local-notebook-path: notebooks/deployments/MainNotebook
-          git-commit: $GITHUB_SHA
```

Comment:
This example was incorrect. Someone from jobs team ran into issue when following this example. Use github context instead: https://docs.github.com/en/actions/learn-github-actions/contexts#github-context